.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
Do you want it to be disallowed altogether? Having it enabled on objects that span multiple lines is pretty nice for diffing stuff.